package/kedro_viz/constants.py

```diff
@@ -25,3 +25,6 @@
         "for users with kedro-datasets >= 2.1.0",
     },
 }
+
+VIZ_SESSION_STORE_ARGS = {"path": ".viz"}
```

Comment:
can we just have one `VIZ_PATH_ARGS` since both are same

Reply:
I too thought of this but since both the constants are for different purposes, though they have the same value for now, it would be nice to keep them separate. Lets say we introduce more keys for `VIZ_METADATA_ARGS` in future other than `path`.